tests/rules/javascript/test.no_unsafe_innerhtml.js

```diff
@@ -1,5 +1,9 @@
 import { VALIDATION_WARNING } from 'const';
 import JavaScriptScanner from 'scanners/javascript';
+import {
+  UNSAFE_DYNAMIC_VARIABLE_ASSIGNMENT,
+  NO_DOCUMENT_WRITE } from 'messages';
```

Comment:
This format looks odd to me but doesn't bother me much. This is what I'm used to but if this is consistent with this project it's cool.

```js

import {
  UNSAFE_DYNAMIC_VARIABLE_ASSIGNMENT,
  NO_DOCUMENT_WRITE,
} from 'messages'; 
// or
import {
  UNSAFE_DYNAMIC_VARIABLE_ASSIGNMENT, NO_DOCUMENT_WRITE,
} from 'messages';